cypress/integration/visitorCanNavigateThroughCategories.feature.js

```diff
@@ -190,10 +197,7 @@ describe("A user visiting the application can display articles by categories", (
         .children()
         .first()
         .within(() => {
-          cy.get("[data-cy=article-created]").should(
-            "contain",
-            "2022-01-13T10:24:38.354Z"
-          );
+          cy.get("[data-cy=article-created]").should("contain", "2022-01-23");
```

Comment:
You can check for the full date

Reply:
not nececary in my opinion. See above answer